cpp/src/cider/function/scalar/ExtensionFunctions.hpp

```diff
@@ -25,8 +25,8 @@
 #include <cmath>
 #include <cstdint>
 #include <cstdlib>
-#include "type/data/CiderTypes.h"
-#include "type/data/funcannotations.h"
+#include "../../type/data/CiderTypes.h"
```

Comment:
absolute path instead?

Reply:
Temporarily use relative path as header can't be found if using absolute path.